src/pkg/packager/create.go

```
@@ -408,7 +422,13 @@ func (p *Packager) addComponent(component types.ZarfComponent) (*types.Component
 			for idx, k := range manifest.Kustomizations {
 				// Generate manifests from kustomizations and place in the package.
 				spinner.Updatef("Building kustomization for %s", k)
-				destination := fmt.Sprintf("%s/kustomization-%s-%d.yaml", componentPath.Manifests, manifest.Name, idx)
+				kname := fmt.Sprintf("kustomization-%s-%d.yaml", manifest.Name, idx)
+				destination := filepath.Join(componentPath.Manifests, kname)
+				if utils.IsURL(k) {
+					dst := filepath.Join(componentPath.Temp, kname)
+					utils.DownloadToFile(k, dst, component.CosignKeyPath)
```

errors are ignored here